### PR TITLE
Created demo user for login and signup page

### LINKED ISSUE #18: FE: Demo User
This ticket does not have a mock-up so please add in the button in the Login and Sign up pages and make sure the design is consistent with the overall look and feel.

- Create a "demo user" functionality that will be used later to allow recruiters and managers to easily test the app's features without having to sign up
- Add a button to allow for signing in as a demo user
- This is effectively a login button with a hardcoded username and password
- Show button both on signup and login pages
- In production you should manually create a user and some test data that would best show off your project. Do this in development right now to test this feature

### PR DESCRIPTION
**Pull request by panwaramita**
### What this PR does (required):
Create a "demo user" functionality that will be used later to allow recruiters and managers to easily test the app's features without having to sign up

Issue Link : Closes #18 

### Screenshots / Videos (front-end only):
Login

signup

### Any information needed to test this feature (required):
Please follow the below steps:
1. In the SignUp page create a user with the following details:
  email: 'hery@gmail.com',
  password: '123456',
name:'hery'
2. Go to the login or signup page and click on the Demo Login button.

### CODE REVIEW

File: server/controllers/auth.js [hunk 1]
```diff
@@ -23,6 +22,7 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
     throw new Error("A user with that username already exists");
   }
 
+  // console.log(req.body);
```
Comment (rajivtitus) [inline server/controllers/auth.js:25]: Please remove logs to the console

File: client/src/pages/Login/LoginForm/DemoLoginButton.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        console.error({ data });
+        setSubmitting(false);
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
+    });
+  };
+  const classes = useStyles();
```
Comment (rajivtitus) [inline client/src/pages/Login/LoginForm/DemoLoginButton.tsx:30]: Nit: Can we move this to line 13 before the function declaration 😄

**Review by rajivtitus — APPROVED** [review]
Excellent work with this Amita! 💯 
Just a minor comment to address before merging

File: client/src/pages/Login/LoginForm/DemoLoginButton.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  return (
+    <Formik
```
Comment (rajivtitus) [inline client/src/pages/Login/LoginForm/DemoLoginButton.tsx:33]: 👍🏽

File: server/controllers/auth.js (hunk 1 above)
Comment (panwaramita) [reply server/controllers/auth.js]: Done the changes

File: client/src/pages/Login/LoginForm/DemoLoginButton.tsx (hunk 2 above)
Comment (panwaramita) [reply client/src/pages/Login/LoginForm/DemoLoginButton.tsx]: Done the changes

**Comment by rajivtitus** [comment]
@panwaramita Thanks for the changes, this is good to merge 👍🏽

**Pull request merged**